junit-platform-suite-api/src/main/java/org/junit/platform/suite/api/SelectMethod.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.suite.api;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE })
+
+public @interface SelectMethod {
```

Comment:
Missing and elsewhere: Javadoc and API Guardian annotation. All public classes should have these. Have a look at the surrounding classes for hints about the style and language to use.

Reply:
API has a default input called `since`, in most select annotations is  `since = 1.10`, but I'm not sure which version I have to type for the annotations I made.

Reply:
The current version of the JUnit Platform is 1.9.3. So the next will be 1.10.

https://github.com/junit-team/junit5#dependency-metadata

Reply:
And it looks like it's 1.10 because some experimental features got made stable. Yours should be marked as experimental. That way we can make breaking changes should we get it wrong.